components/slack/sources/new-message-in-channels/new-message-in-channels.js

```diff
@@ -1,14 +1,17 @@
-const slack = require('https://github.com/PipedreamHQ/pipedream/components/slack/slack.app.js')
+const slack = require('../../slack.app.js')
 
 module.exports = {
-  name: "Slack - New Message In Conversation(s)",
+  key: "slack-new-message-in-channels",
```

Comment:
Maybe keep the "New Message in Conversation" format, since this lets you listen for messages in group chats as well as channels, I believe? May want to confirm with Dan.

Reply:
just closing the loop. made a call to leave as it. thanks!